markdown/en/ff-loading-spinner.md

```diff
@@ -0,0 +1,74 @@
+## Overview
+
+`ff-loading-spinner` is a leightweight spinning loading indicator that is 
+triggered by common events fired by FACT-Finder Web Components. It uses 
+inexpensive CSS-Animations and does not trigger `layout` or `paint` 
+operations in Blink and Gecko-based browsers.
+
+The element gets activated by the following events:
+- search
+- filter
+- clearFilter
+- sort
+- advisor
+
+It is deactivated when the action is completed.
+
+**Example:** When a search request is sent via the `ff-searchbox` the spinner
+is activated. As soon as FACT-Finder's response is received, the spinner is 
+deactivated.
+
+## Usage 
+
+Assuming you have integrated `ff-communication` and `ff-searchbox` you simply
+need to place the `ff-loading-spinner` element in your HTML.
+
+```html
+<ff-loading-spinner></ff-loading-spinner>
+```
+As soon as a request is sent to FACT-Finder, `ff-loading-spinner` will
+be displayed. When the response is received the element will be hidden again.
+
+#### Changing color
+
+You can also change the element's  color by passing a CSS color value.
```

Comment:
Probably irrelevant when rendered. But looks like there is a double space before `color`.